botbot_plugins/plugins/jira.py

```diff
@@ -0,0 +1,65 @@
+import requests
+import json
+from urlparse import urljoin
+from .. import config
+from ..base import BasePlugin
+from ..decorators import listens_to_all, listens_to_mentions
+
+
+
+class Config(config.BaseConfig):
+    jira_link = config.Field(help_text="Jira Link, eg: 'https://tickets.metabrainz.org'")
```

Comment:
Being nitpicky again but I believe "URL" would be more appropriate than "link" here (both in variable name and help text).